tests/tests-core/src/main/java/com/gentics/mesh/search/LanguageOverrideSearchTest.java

```diff
@@ -182,6 +189,50 @@ public void testLanguageOverride() {
 		assertPublishedContentCount();
 	}
 
+	@Test
+	public void testIndexSyncWithMicronodeField() throws Exception {
+		MicroschemaResponse microschema = createMicroschema("micro");
+		call(() -> client().assignMicroschemaToProject(PROJECT_NAME, microschema.getUuid()));
+		SchemaResponse schema = createSchema(loadResourceJsonAsPojo("schemas/languageOverride/pageWithMicronode.json", SchemaCreateRequest.class));
+		createContent();
+		waitForSearchIdleEvent();
+
+		String microschemaHash = tx(tx -> {
+			HibSchema schema1 = tx.schemaDao().findByUuid(schema.getUuid());
+			HibSchemaVersion version = schema1.getLatestVersion();
+			return version.getMicroschemaVersionHash(initialBranch());
+		});
+
+		assertDocumentCount(fromEntries(
+			docs("de", DRAFT, 2),
+			docs("fr", DRAFT, 1),
+			docs("zh", DRAFT, 1),
+			docs("ja", DRAFT, 1),
+			docs("ko", DRAFT, 1),
+			// italian and english contents use default settings
+			docs(microschemaHash.toLowerCase(), DRAFT, 3)
+		));
+
+		SyncEventHandler.invokeSyncCompletable(meshApi()).blockingAwait(30, TimeUnit.SECONDS);
+
+		assertDocumentCount(fromEntries(
```

Comment:
Not sure if this will work, as the content of the indices is cleared _during_ the sync.
In the test, you'd need always need to be "lucky" to get the timing right I think?

Reply:
This is how it works in all the tests around, just wrapped by `assertContentCount()`.
Since `invokeSyncCompletable()` is awaited for 30 seconds the most (which has to be enough for 1 page case), no race is expected.

Reply:
Also, the original usecase is reproducible with this test, if the fix is removed.